.github/ISSUE_TEMPLATE/new-repo-checklist.md

```diff
@@ -58,8 +58,7 @@ https://github.com/maplibre/[repo]
   *For smaller projects it might be OK to ignore this rule.*
 
 ## Miscelaneous
-- [ ] The repo must not have any personal branches.
-  *All work should be done on forks and submitted via PRs, including by the admins.*
+- [ ] The repo owner(s) must make sure branches aren't staking up, a good way to solve this is to encourage the usage of forks and PRs instead of private branches
```

Comment:
```suggestion
- The repo owner(s) must make sure branches aren't staking up, a good way to solve this is to encourage the usage of forks and PRs instead of private branches
```

Since this is not actionable I would just make this a bullet point instead of a todo.